glide.lock

```diff
@@ -1303,7 +1310,7 @@ imports:
 - name: gopkg.in/yaml.v2
   version: 53feefa2559fb8dfa8d81baad31be332c97d6c77
 - name: k8s.io/api
-  version: 78d52114c359511e7d0d5af4fc50148b5392d7b4
+  version: 30a449993d4d38589fa79362431bd9b04644cab5
```

Comment:
@deads2k this is probably OK, reflecting the branches that are now synced by publishing bot.